web-components/.eslintrc.js

```diff
@@ -7,6 +7,7 @@ module.exports = {
     {
       files: ['*.tsx', '*.ts'],
       rules: {
+        'react-hooks/rules-of-hooks': 'off',
```

Comment:
should we just turn this off for the new lines in the stories that cause the issue?